dev_scripts/cbtest.py

```diff
@@ -0,0 +1,131 @@
+import json
+import pyperclip
+from datasets.chapterbreak import ChapterBreakDataset, split_in_pages
+from utils.openai import ask_llm, make_system_message, make_user_message
+from utils.tokens import token_len
+
+
+total_cost = 0
+model = "gpt-3.5-turbo-0125"
+ds = ChapterBreakDataset(split="ao3")
+samples = ds.get_samples(ds.load_data())
+s = samples[1]
+llm = lambda ctx: ask_llm(ctx, model, temperature=0, context_length=16384, cost_callback=cost_callback)
```

Comment:
Isn't gpt-3.5-turbo only 4k context? Won't this break?

Reply:
Ah right its 16k, I see below

Reply:
Yeah, at some point they unified the nomenclature, so older versions have a 4k context and the newest ones 16k.